docs/readmes/orc8r/upgrade_1_4.md

```diff
@@ -78,11 +78,23 @@ terraform refresh
 
 ## Terraform apply
 
+Terraform the upgrade. We'll also need to manually rescale the Prometheus
+deployment, as a workaround for quirks in how its deployment handles upgrading.
```

Comment:
minor nit: Can you reference the prometheus crashlooping issue here or mention it, so that users won't be confused on why they might have to do this additionally.